WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -372,6 +372,9 @@ private void endMigration() {
 
     public static void registerMigrationListener(MigrationListener listener) {
         sMigrationListener = listener;
+        if (sIsMigrationError) {
+            sMigrationListener.onError();
```

Comment:
I'm not sure why a null check would necessary? `sMigrationListener` is set as the `listener` param right above, which is not going to be `null` (we could annotate the param with `@NonNull`).